castle/validators/not_supported.py

```diff
@@ -0,0 +1,10 @@
+from castle.exceptions import InvalidParametersError
+
+
+class ValidatorsNotSupported(object):
+
```

Comment:
Empty line

Reply:
@nijikon - I used autopep8 for empty lines styling